src/components/Notifications/Notifications.module.css

```
@@ -1,27 +1,27 @@
-    width: 0;
-    height: 0;
-    border-bottom: 15px solid #fff;
-    border-bottom: 15px solid #fff;
-    border-left: 15px solid transparent;
-    top: -10px;
-    right: 165px;
+  content: '';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-bottom: 15px solid #fff;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "border-bottom" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)